api/attribute.proto

```diff
@@ -400,6 +400,13 @@ message FourOctetAsSpecificExtended {
   uint32 local_admin = 4;
 }
 
+message LinkBandiwdthExtended {
+  bool is_transitive = 1;
+  uint32 sub_type = 2;
```

Comment:
How `is_transitive` and `sub_type` is used?

Reply:
Fixed.